app/ledger.js

```diff
@@ -143,15 +143,14 @@ const doAction = (action) => {
 
         case settings.MINIMUM_VISIT_TIME:
           if (action.value <= 0) break
-
-          synopsis.options.minDuration = action.value * 1000
```

Comment:
in order for the ledger code to actually do the pruning, you'll need this line. just remove the "* 1000"